pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -2505,12 +2507,29 @@ func TestWorkloadInstanceHandler_WorkloadInstanceIndex(t *testing.T) {
 	verifyGetByIP("2.2.2.2", []*model.WorkloadInstance{wi1})
 	verifyGetByIP("3.3.3.3", []*model.WorkloadInstance{wi2})
 
+	wiWithMulAddrs := &model.WorkloadInstance{
```

Comment:
multiple addresses do not seem to be verified?

Reply:
Please check below code at L#2525